HW_29/task_2.py

```diff
@@ -0,0 +1,34 @@
+import requests
+def get_coordinates(city):
+    url = f"https://api.open-meteo.com/v1/geocode?query={city}"
+    response = requests.get(url)
+    data = response.json()
+    if "latitude" in data and "longitude" in data:
+        return data["latitude"], data["longitude"]
+    return None, None
```

Comment:
nit: use two blank lines between functions
```suggestion


def get_coordinates(city):
    url = f"https://api.open-meteo.com/v1/geocode?query={city}"
    response = requests.get(url)
    data = response.json()
    if "latitude" in data and "longitude" in data:
        return data["latitude"], data["longitude"]
    return None, None
   
   
```